// @polkadot/ui-util/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
Object.defineProperty(exports, "formatBalance", {
  enumerable: true,
  get: function get() {
    return _util.formatBalance;
  }
});
Object.defineProperty(exports, "formatDecimal", {
  enumerable: true,
  get: function get() {
    return _util.formatDecimal;
  }
});
Object.defineProperty(exports, "formatElapsed", {
  enumerable: true,
  get: function get() {
    return _util.formatElapsed;
  }
});
Object.defineProperty(exports, "formatNumber", {
  enumerable: true,
  get: function get() {
    return _util.formatNumber;
  }
});
Object.defineProperty(exports, "calcSi", {
  enumerable: true,
  get: function get() {
    return _util.calcSi;
  }
});
Object.defineProperty(exports, "findSi", {
  enumerable: true,
  get: function get() {
    return _util.findSi;
  }
});
Object.defineProperty(exports, "isTestChain", {
  enumerable: true,
  get: function get() {
    return _util.isTestChain;
  }
});

var _util = require("@polkadot/util");

// Copyright 2017-2019 @polkadot/ui-util authors & contributors
// This software may be modified and distributed under the terms
// of the Apache-2.0 license. See the LICENSE file for details.
console.warn('@polkadot/ui-util has been incorporated into @polkadot/util (with the same export names)');
